source/d3d11/d3d11_device_context.cpp

```diff
@@ -1,12 +1,16 @@
 /**
- * Copyright (C) 2014 Patrick Mours. All rights reserved.
- * License: https://github.com/crosire/reshade#license
- */
+* Copyright (C) 2014 Patrick Mours. All rights reserved.
+* License: https://github.com/crosire/reshade#license
+*/
```

Comment:
Please make sure whitespace matches that of the original code. There are quite a few of those issues, so I'm not going to mark them all. See the pull request diff.